hack/terraform/aws/delete_dependency.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Delete security groups on AWS created by users.
+vpc_id="$( terraform output vpc_id | sed 's/\"//g' )"
+region="$( terraform output region | sed 's/\"//g' )"
+
+sgs="$( aws ec2 describe-security-groups --region $region --filters Name=vpc-id,Values=$vpc_id | jq '.SecurityGroups[].GroupId' )"
```

Comment:
sgs will list sgs for a vpc, nephe_SG for terraform created SG, what is vpc_SG used for ??

Reply:
@reachjainrahul  
sgs: all the SG aws command detected
vpc_sg: SG in VPC
nephe_SG is other SGs seems like they are with the VMs, they are in the different output part, we need detect them in defferent ways

Reply:
@reachjainrahul already changed it to default_VM_SG